Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -16,7 +16,7 @@
     <% if !root_url.start_with?(HOMEBREW_BOTTLE_DEFAULT_DOMAIN) %>
     root_url "<%= root_url %>"
     <% end %>
-    <% if ![Homebrew::DEFAULT_PREFIX, "/usr/local"].include?(prefix) %>
+    <% if ![Homebrew::HOMEBREW_DEFAULT_PREFIX, Homebrew::LINUXBREW_DEFAULT_PREFIX].include?(prefix) %>
```

Comment:
These are no longer declared in the `Homebrew` module

Reply:
Fixed.